web/src/app/projects/edit/edit.component.html

```diff
@@ -38,7 +38,7 @@
             <mat-divider></mat-divider>
             <mat-card-actions>
                 <div fxLayout="row" fxLayoutGap="20px" fxLayoutAlign="center center">
-                    <button mat-stroked-button type="button" [routerLink]="['/project', uid]">Cancel</button>
+                    <button mat-stroked-button type="button" (click)="onClickCancel()">Cancel</button>
```

Comment:
Does this not add more code, rather than just having a `routerLink`?

Reply:
initially I was facing issue with relative link URL in routerLink so I have updated it click event. Now I found that we need to just go on its parent route when click on cancel so updated it again with routerLink ='../'

Thanks for pointing it out.

Reply:
I am not sure about `../`, let me have a think 🤔

Reply:
Can we not go from the root of the `uri` by prepending it with `/`?

What is the url its going `from` and `to`?

Reply:
It means its going one path up.
It is going from projects/:uid/edit to projects:uid .

Reply:
Ok I will have a look, because I would have thought using `/projects/:uid` would work.

Lets merge it for now, the rest of it looks fine and I don't want to block you. I can do some research later on 🤓

Reply:
When I was on route "/projects/:uid/edit" and I click on cancel button using routerLink = ["projects", ''uid''] then it was appending this path in current route path which is wrong. Ideally we have to go one path up which can be achieved using '../'

Please let me know what do you think on this ?